src/handlers/wildcard/unassign.ts

```diff
@@ -49,7 +51,18 @@ const checkBountyToUnassign = async (issue: Issue): Promise<boolean> => {
   const curTimestamp = new Date().getTime();
   const lastActivity = await lastActivityTime(issue, comments);
   const passedDuration = curTimestamp - lastActivity.getTime();
+  const pullRequest = await getOpenedPullRequestsForAnIssue(issue.number, issue.assignee);
 
+  const { data: reviews } = await context.octokit.pulls.listReviews({
+    owner: payload.repository.owner.login,
+    repo: payload.repository.name,
+    pull_number: pullRequest[0].id,
+  });
```

Comment:
I have been consistently asking hunters to move helper fns using octokit library to src/helpers. we have tons of helper functions defined there. please create a helper function for this block and use that function here.